HTTP2-Core.package/H2CacheEntry.class/class/expireDuration.st

```diff
@@ -0,0 +1,4 @@
+constants
+expireDuration
+
+	^ 1 day
```

Comment:
should be string or is this a defined datatype?

Reply:
defined datatype (Duration i think)